cookbooks/travis_build_environment/recipes/ibm_advanced_tool_chain.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+def installation_allowed?
+  %w[bionic focal xenial].include?(node['lsb']['codename']) && node['kernel']['machine'] == 'amd64'
```

Comment:
You choose OS in packer-templates and another condition is not needed.

Reply:
Done